bin/make-consensus.py

```diff
@@ -97,16 +98,16 @@ def main():
 
         e.execute("bcftools index '%s'" % vcfFile)
 
-    if args.maskNoCoverage:
+    if args.maskLowCoverage and args.maskLowCoverage not < 0:
```

Comment:
Why not just say `if args.maskLowCoverage > 0` ?    (Also I suspect `args.maskLowCoverage not < 0` is not valid Python, though I could be wrong.)